lms/static/js/spec/learner_dashboard/certificate_view_spec.js

```diff
@@ -50,15 +50,13 @@ define([
             });
 
              it('should display no certificate box if certificates list is empty', function() {
-                var $certificate;
                 view.remove();
                 setFixtures('<div class="certificates-list"></div>');
                 view = new CertificateView({
                     context: {certificatesData: []}
                 });
                 view.render();
-                $certificate = view.$el.find('.certificate-link');
-                expect($certificate.length).toBe(0);
+                expect(view.$('.certificates-list').length).toBe(0);
```

Comment:
I am checking that @rlucioni, it is the ```.certificate-list``` element and if its length is zero it isn't there.